examples/runtime-example/README.md

```diff
@@ -27,21 +27,66 @@ Then developers can create runtime instance:
 
 See [`./runtime.ts`](https://github.com/aerogear/graphback/blob/master/examples/runtime-example/src/runtime.ts#L32) for a fully functional example.
 
-### Running example
+### Running example (postgres)
 
 The project has been created using `graphback`. Run the project using the following steps. 
 - Start the database
 ```
 docker-compose up -d
 ```
 
-- create database
+- Define your schema in the `model/runtime.graphql` file. Or use the default:
+
 ```
-graphback db
+type User {
+  id: ID!
+  name: String
+}
 ```
+
 - Start the server
 ```
-npm start
+npm run develop
+```
+
+Or, if using yarn
+
+```
+yarn develop
+```
+
+### Running example (sqlite)
+
+The project has been created using `graphback`. Run the project using the following steps. 
+- Modify the db json object in the `graphback.json` config file. For an in-memory database, use the below config as-is, alternatively replace `:in-memory:` with the desired filename.
+```
+"db": {
+  "dbConfig": {
+    "filename": ":in-memory:"
+  },
+  "database": "sqlite3"
+},
 ```
 
-Enjoy runtime app
+- Define your schema in the `model/runtime.graphql` file. Or use the default:
+
+```
+type User {
+  id: ID!
+  name: String
+}
+```
+
+- Start the server (for the first time)
+```
+npm run develop
+```
+Or, if using yarn
+```
+yarn develop
+```
+
+If the server is being re-run, modify the `src/runtime.ts` and comment out the `migrateDb` function, since it will not be possible to re-migrate the database with sqllite3.
```

Comment:
Also we need to change the interface from `PGKnex` to `Knex`

Reply:
Updated this too